protocol/src/main/java/org/opensearch/sql/protocol/response/format/FlatResponseFormatter.java

```diff
@@ -30,6 +30,10 @@ public FlatResponseFormatter(String seperator, boolean sanitize) {
     this.sanitize = sanitize;
   }
 
+  public String getFormat() {
+    return "plain/text; charset=UTF-8";
```

Comment:
need a unit test. 

Also, move this to a constant (`public static final String`)

Reply:
added unit test in https://github.com/Bit-Quill/opensearch-project-sql/pull/279/commits/0df2a1b34b182cea23bfb490d7143bd726aa00d4